python/pyspark/sql/readwriter.py

```diff
@@ -659,7 +659,7 @@ def text(self, path, compression=None):
         self._jwrite.text(path)
 
     @since(2.0)
-    def csv(self, path, mode=None, compression=None, sep=None, quote=None, escape=None,
+    def csv(self, path, mode=None, compression=None, sep=None, encoding=None, quote=None, escape=None,
```

Comment:
We need to place this new option at the end. Otherwise, it will breaks existing codes that use this options as a positional argument (not keyword argument).